contracts/interfaces/0.8.x/IGenArt721CoreContractV3_Engine_Flex.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+import "./IAdminACLV0.sol";
+import "./IGenArt721CoreContractV3_Engine.sol";
+
+interface IGenArt721CoreContractV3_Engine_Flex is
+    IGenArt721CoreContractV3_Engine
+{
+    // preferredIPFSGateway is a url string
+    function preferredIPFSGateway() external view returns (string memory);
+
+    // preferredArweaveGateway is a url string
+    function preferredArweaveGateway() external view returns (string memory);
+
+    event ExternalAssetDependencyUpdated(
+        uint256 indexed _projectId,
+        uint256 indexed _index,
+        string _cid,
```

Comment:
in natspec comments, can we explicitly call out that this is the empty string if _cid is on-chain data?